src/pages/billing/SpendReport/SpendReport.ts

```diff
@@ -179,29 +230,30 @@ export const SpendReport = ({ tab, billingProject }) => {
           div({ style: { gridRowStart: 1, gridColumnStart: 1 } }, [
             h(IdContainer, [id => h(Fragment, [
               h(FormLabel, { htmlFor: id }, ['Date range']),
-              h(Select, {
+              h(NumberSelect, {
```

Comment:
You can use the parameterized component directly by putting it in parenthesis:
```typescript
h((Select<number>), { ...
```
The hyperscript helpers confuse the syntax of react components.

Reply:
Thanks!